frontend/src/common/types/billing/billing-withdraw-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type BillingWithdrawDto = {
+  usersCurrentBalance: number;
```

Comment:
```suggestion
  userCurrentBalance: number;
```

Reply:
I thought about proper grammatic, but agree, here seems to be confusing